src/main/java/mate/academy/internetshop/controller/MainMenu.java

```diff
@@ -6,10 +6,17 @@
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.lib.Inject;
+
 public class MainMenu extends HttpServlet {
+    @Inject
+    private static ItemDao itemDao;
+
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        itemDao.get(1L);
```

Comment:
This is for test, I suppose. please, remove

Reply:
ok